pkg/kubernetes/cluster.html

```diff
@@ -235,6 +235,47 @@ <h4 class="modal-title" translatable="yes">Adjust Service</h4>
       </div>
     </div>
 
+    <div class="modal" id="adjust-entity" type="text/ng-template" tabindex="-1" role="dialog" data-backdrop="static">
+      <div class="modal-dialog modal-sm">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h4 class="modal-title" translatable="yes">Adjust Replication Controller</h4>
+          </div>
+          <div class="modal-body">
+            <table class="cockpit-form-table">
+              <tr class="adjust-replicas-header">
+                <td colspan="2" class="header">Replicas</td>
+                <td>
+                    <input class="form-control adjust-replica" type="text" id="replica-count" value="">
+                </td>
+              </tr>
+            </table>
+          </div>
+          <div class="modal-footer">
+            <button class="btn btn-default" translatable="yes" data-dismiss="modal">Cancel</button>
+            <button class="btn btn-primary" translatable="yes" id="adjust-rc-button">Adjust</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="modal" id="delete-pod-dialog" type="text/ng-template" tabindex="-1" role="dialog" data-backdrop="static">
```

Comment:
This seems to be a duplicate of the code below. Are you sure we need mulitple copies?